src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -81,7 +75,6 @@ export const RoomModal: React.FC<RoomModalProps> = ({ onHide, room }) => {
               </div>
               <div className="col">
                 <RoomModalOngoingEvent
-                  roomTitle={room.title}
```

Comment:
Guessing this was just never used/needed?

Reply:
It was used in `tracking user's location` logic before. Have refactored that

Reply:
Makes sense.